meta/Vertices.m

```diff
@@ -693,6 +748,20 @@ are scalars without (generation) indices.  This alone does not yet
 ReplaceUnrotatedFields[SARAH`Cp[p__][lorentz_]] :=
     ReplaceUnrotatedFields[SARAH`Cp[p]][lorentz];
 
+IsNonZeroVertex[fields_List, vertexList_:{}, useDependences_:False] :=
+    Module[{sortedFields, cached, vertex, isNonZero},
```

Comment:
`isNonZero` seems to be unused

Reply:
It's used in the new test. The reason I've added it to `Vertices` is that's where decays will need it to be.

Reply:
Sorry, I don't understand. Isn't `isNonZero` a variable that is local to the module and thus not visible externally?

Reply:
Oh, sorry. I thought you meant the function itself. I blame Dylan ;) Yes, it looks like this variable is unused. I'll remove it.